Makefile

```diff
@@ -0,0 +1,32 @@
+# shamelessly adapt https://github.com/qiime2/q2-emperor/blob/master/Makefile
+.PHONY: all lint test test-cov install dev clean distclean
+
+PYTHON ?= python
+
+all: ;
+
+lint:
+	flake8 microsetta_public_api
+
+test: all
+	py.test
+
+test-install: all
+	# ensure the package is installed and the app is buildable. this test
+	# is a passive verification that non-py essential files are part of the
+	# installed entity.
+	cd /  # go somewhere to avoid relative imports
+	python -c "from microsetta_public_api import server; server.build_app()"
+
+test-cov: all
+	py.test --cov=microsetta_public_api
```

Comment:
Need to add `pytest-cov` to the `extras_require["dev"]` in `setup.py` if you want to use the coverage flag

Reply:
good spot, added that and flake8 to the pip reqs file

Reply:
You also removed the extras from the `setup.py`

Can you revert that change and use the `pip install .[dev]`? Or otherwise install `pytest`, `pytests-cov`, and `flake8` in a way that does not intermix them with the run dependencies?

I intentionally wanted to keep the run and test dependencies separate.

Reply:
The intention was to make this consistent with the other microsetta repos, and to increase consistency with other biocore and QIIME 2 repos. The expense of these well maintained dependencies is small... I'm not sure I understand the value in deviating here

Reply:
Alrighty then guess its fine